test/unit/forms/declaration/ExporterDetailsSpec.scala

```diff
@@ -25,12 +25,6 @@ class ExporterDetailsSpec extends WordSpec with MustMatchers with LightFormMatch
 
   import forms.declaration.ExporterDetailsSpec._
 
-  "Exporter Details" should {
-    "have correct id" in {
-      ExporterDetails.id mustEqual "ExporterDetails"
-    }
-  }
-
```

Comment:
Why is this removed if in IsExcSpec you are testing the same thing?

Reply:
We're not using the ExporterDetails Id anywhere. Removed ID and test for it

Reply:
IsExs tests key, not page ID because we're not using page id anymore